gateway/ln-gateway/src/db.rs

```diff
@@ -1,13 +1,15 @@
 use fedimint_core::config::FederationId;
 use fedimint_core::encoding::{Decodable, Encodable};
 use fedimint_core::{impl_db_lookup, impl_db_record};
+use fedimint_ln_common::LightningGateway;
 use lightning::routing::gossip::RoutingFees;
 use secp256k1::KeyPair;
 
 #[repr(u8)]
 #[derive(Clone, Debug)]
 pub enum DbKeyPrefix {
     FederationConfig = 0x04,
+    Registration = 0x05,
```

Comment:
nit: `RegisterFederation` or `FederationRegister`? Just looking for a prefix which reminds me of  registration scenario

Reply:
I opted for `FederationRegistration`